en/django_urls/README.md

```diff
@@ -93,10 +93,23 @@ The last part, `name='post_list'`, is the name of the URL that will be used to i
 
 If you try to visit http://127.0.0.1:8000/ now, then you'll find some sort of 'web page not available' message. This is because the server (remember typing `runserver`?) is no longer running. Take a look at your server console window to find out why.
 
-![Error](images/error1.png)
+{% filename %}{{ warning_icon }} command-line{% endfilename %}
+
+```shell
+    return _bootstrap._gcd_import(name[level:], package, level)
+  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
+  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
+  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
+  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
+  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
+  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
+  File "/Users/dana/Codes/djangogirls/blog/urls.py", line 5, in <module>
```

Comment:
Typically the name `ola` is used
```suggestion
  File "/Users/ola/Codes/djangogirls/blog/urls.py", line 5, in <module>
```

Reply:
What about the rest of the path? Do you have any convention? Preferences about it prioritizing Windows/OSX/Linux?

Reply:
If you look at https://tutorial.djangogirls.org/en/django_start_project/ it's in `~/djangogirls`. I'm not sure about which OS to prioritize.

Reply:
Noted, let's go with `/Users/ola/djangogirls/...` for now!